policy/scan/local_scanner.go

```diff
@@ -979,29 +998,32 @@ func (s *localAssetScanner) ensureBundle() error {
 	return err
 }
 
-func (s *localAssetScanner) runPolicy() (*policy.Bundle, *policy.ResolvedPolicy, error) {
+func (s *localAssetScanner) runPolicy() (*policy.ResolvedPolicy, error) {
 	var hub policy.PolicyHub = s.services
 	var resolver policy.PolicyResolver = s.services
 
-	log.Debug().Str("asset", s.job.Asset.Mrn).Msg("client> request policies bundle for asset")
-	assetBundle, err := hub.GetBundle(s.job.Ctx, &policy.Mrn{Mrn: s.job.Asset.Mrn})
-	if err != nil {
-		return nil, nil, err
+	// If we run in debug mode, download the asset bundle and dump it to disk
+	if val, ok := os.LookupEnv("DEBUG"); ok && (val == "1" || val == "true") {
+		log.Debug().Str("asset", s.job.Asset.Mrn).Msg("client> request policies bundle for asset")
+		assetBundle, err := hub.GetBundle(s.job.Ctx, &policy.Mrn{Mrn: s.job.Asset.Mrn})
+		if err != nil {
+			return nil, err
+		}
+		log.Debug().Msg("client> got policy bundle")
+		logger.TraceJSON(assetBundle)
+		logger.DebugDumpYAML("assetBundle", assetBundle)
 	}
-	log.Debug().Msg("client> got policy bundle")
-	logger.TraceJSON(assetBundle)
-	logger.DebugDumpJSON("assetBundle", assetBundle)
 
 	rawFilters, err := hub.GetPolicyFilters(s.job.Ctx, &policy.Mrn{Mrn: s.job.Asset.Mrn})
```

Comment:
we should probably dump `rawFilters` to a file since that is what actually gets used

Reply:
added